Gemfile

```
@@ -69,6 +69,7 @@ gem 'subprocess', require: false
 gem 'uglifier', '~> 4.2'
 gem 'user_agent_parser'
 gem 'valid_email', '>= 0.1.3'
+gem 'view_component', '~>2.40.0', require: 'view_component/engine'
```

supernit for consistency

```suggestion
gem 'view_component', '~> 2.40.0', require: 'view_component/engine'
```

---

> supernit for consistency

Nice catch, thanks.